requirements.txt

```diff
@@ -1,4 +1,5 @@
 requests
 PySocks
 cryptography
-pyyaml
\ No newline at end of file
+pyyaml
+pycurl==7.43.0
```

Comment:
why hardcode this value?

Reply:
`AppVeyor` was failing before I set it

Reply:
it seems like latest version dont have any wheels  (binary files) for latest version, we should skip that version that version `pycurl != 7.43.0.3`.  there is a ticket for it at https://github.com/pycurl/pycurl/issues/569